examples/support/mock_data.py

```diff
@@ -204,24 +287,39 @@ def integrated_pzfxn(zmax, func):
     return GCData([zsrc_list, zsrc_list], names=('ztrue', 'z'))
 
 
-def _compute_photoz_pdfs(galaxy_catalog, photoz_sigma_unscaled, ngals):
-    r"""Add photo-z errors and PDFs to the mock catalog."""
+def _compute_photoz_pdfs(galaxy_catalog, photoz_sigma_unscaled):
+    """Private function to add photo-z errors and PDFs to the mock catalog.
+    
+    Parameters
+    ----------   
+    galaxy_catalog : clmm.GCData
+        Input galaxy catalog to which photoz PDF will be added
+    photoz_sigma_unscaled : float
+        Width of the Gaussian PDF, without the (1+z) factor
+
+    Returns
+    -------
+    galaxy_catalog : clmm.GCData
+        Output galaxy catalog with columns corresponding to the bins 
+        and values of the redshift PDF for each galaxy.
+    """
     galaxy_catalog['pzsigma'] = photoz_sigma_unscaled*(1.+galaxy_catalog['ztrue'])
     galaxy_catalog['z'] = galaxy_catalog['ztrue'] + \
-                          galaxy_catalog['pzsigma']*np.random.standard_normal(ngals)
+                          galaxy_catalog['pzsigma']*np.random.standard_normal(len(galaxy_catalog))
+
     pzbins_grid, pzpdf_grid = [], []
     for row in galaxy_catalog:
-        zmin, zmax = row['ztrue'] - 0.5, row['ztrue'] + 0.5
+        zmin, zmax = row['z'] - 0.5, row['z'] + 0.5
```

Comment:
This would cause the (redshift) interval (for the probability) to be different for different galaxies. But maybe it's okay.

Reply:
I confirm this is not a problem.